src/server.jl

```diff
@@ -39,15 +39,26 @@ function ws_handler(app::App)
   return handler
 end
 
-const default_port = 8000
-
-function serve(s::Server, port = default_port)
-  @async @errs run(s, port)
+function serve(s::Server; args...)
```

Comment:
This looks like it'd be a lot simpler as `serve(s::Server, port = 8000; ssl = nothing)`